testutil/blockio.go

```diff
@@ -0,0 +1,99 @@
+package testutil
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/decred/dcrd/wire"
+)
+
+// BlockFilename generates file name for a Block
+func BlockFilename(index uint64) string {
+	return "block-" + padNumberWithZero(index, 10) + ".bin"
```

Comment:
are you planning to use `padNumberWithZero` often?  I would just have said:

`return fmt.Sprintf("block-%010d.bin", index)`

Simpler, shorter, more readable.